src/components/Editor/Resources/ResourceSeatingCapacity.vue

```diff
@@ -22,34 +22,36 @@
 
 <template>
 	<div class="resource-capacity">
-		<div class="resource-capacity__input">
-			<input type="number"
-				min="0"
-				:placeholder="placeholder"
-				:value="value"
-				@input.prevent.stop="changeValue">
-		</div>
+		<NcTextField :label="placeholder"
+			type="number"
+			min="0"
+			:placeholder="placeholder"
+			:value.sync="capacityValue" />
 	</div>
 </template>
 
 <script>
+import NcTextField from '@nextcloud/vue/dist/Components/NcTextField.js'
 export default {
 	name: 'ResourceSeatingCapacity',
+	components: {
+		NcTextField,
+	},
 	props: {
 		value: {
 			type: Number,
 			required: true,
 		},
 	},
+	data() {
+		return {
+			capacityValue: this.value,
+		}
+	},
```

Comment:
Local state is not required and it should use the value prop and emit an event on change. Please also see my other comment.

Reply:
Done